packages/trader/src/Modules/MT5/Components/mt5-real-account-display.jsx

```diff
@@ -47,17 +47,27 @@ const MT5RealAccountDisplay = ({
     toggleAccountsDialog,
     toggleShouldShowRealAccountsList,
     can_have_more_real_synthetic_mt5,
+    residence_list,
 }) => {
     const should_show_trade_servers = is_logged_in && !is_eu && has_real_account && can_have_more_real_synthetic_mt5;
     const [active_hover, setActiveHover] = React.useState(0);
 
-    const has_required_credentials =
-        account_settings.citizen && account_settings.tax_identification_number && account_settings.tax_residence;
+    const has_required_credentials = () => {
```

Comment:
```suggestion
    const hasRequiredCredentials = () => {
```
Function names should be in camelCase

Reply:
thanks Bala, Updated